.gitignore

```diff
@@ -124,3 +124,38 @@ env/
 
 ### Log files ###
 *.log
+
+# Local .terraform directories
+**/.terraform/*
+
+# .tfstate files
+*.tfstate
+*.tfstate.*
+
+# Crash log files
+crash.log
+crash.*.log
+
+# Exclude all .tfvars files, which are likely to contain sensitive data, such as
+# password, private keys, and other secrets. These should not be part of version 
+# control as they are data points which are potentially sensitive and subject 
+# to change depending on the environment.
+*.tfvars
+*.tfvars.json
+
+# Ignore override files as they are usually used to override resources locally and so
+# are not checked in
+override.tf
```

Comment:
Not important here, but a fact I like to pull out any time I see copy-paste between .gitignore & .dockerignore is that they do have slightly different syntaxes - [see this article for more info](https://zzz.buzz/2018/05/23/differences-of-rules-between-gitignore-and-dockerignore/) as it can catch you out

In particular, your .gitignore is ignoring all files called override.tf whereas your .dockerignore is only ignoring files in the root folder with that name